install_config/install/advanced_install.adoc

```diff
@@ -681,6 +681,154 @@ openshift_examples_modify_imagestreams=true
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+----
+
+. (Optional) Include any of the following role variables in the `[OSEv3:vars]`
+section you wish to change:
++
+----
+[OSEv3:vars]
+openshift_storage_glusterfs_namespace=glusterfs <1>
+openshift_storage_glusterfs_name=storage <2>
+----
+<1> The project (namespace) to host the storage pods. Defaults to `glusterfs`.
+<2> A name to identify the GlusterFS cluster, which will be used in resource names.
+Defaults to `storage`.
+
+. Add a `[glusterfs]` section with entries for each storage node that will host
+the GlusterFS storage and include the `glusterfs_devices` list parameter in the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs] 
+192.168.10.11 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+192.168.10.12 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+192.168.10.13 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
++
+The `glusterfs_devices` list of raw block devices will be completely managed as
+part of a GlusterFS cluster. There must be at least one device listed. Each
+device must be bare, with no partitions or LVM PVs.
+
+. Additional steps must be run manually after the cluster installation has
+completed. Continue with the rest of the preparation of your inventory file,
+then see xref:running-the-advanced-installation[Running the Advanced Installation] for the additional steps to complete the configuration of
+containerized GlusterFS persistent storage.
+
+[[advanced-install-configuring-openshift-container-registry]]
+=== Configuring the OpenShift Container Registry
+
+Additional configuration options are available at installation time for the
+xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[OpenShift Container Registry].
+
+[[advanced-install-glusterfs-backed-registry]]
+==== Configuring a GlusterFS-Backed Registry
+
+Similar to
+xref:advanced-install-containerized-glusterfs-persistent-storage[configuring containerized GlusterFS for persistent storage], a containerized
+GlusterFS-backed OpenShift Container Registry can be configured and deployed
+during the initial installation of the cluster to offer a more reliable
+container registry.
+
+If no registry storage options are used, the default {product-title} registry is
+ephermal and all data will be lost if the pod no longer exists. {product-title}
+also supports a single node NFS-backed registry, but this option lacks
+redundancy and reliability compared with the GlusterFS-backed option.
+
+To configure a containerized GlusterFS-backed registry:
+
+. Specific prerequisites and host preparation are required to run this deployment.
+See
+xref:../../install_config/install/prerequisites.adoc#prereq-containerized-glusterfs-considerations[Containerized GlusterFS Considerations] and ensure the steps have been satisfied.
```

Comment:
Remove the numbering from the above paragraph, then add:

```
Configuration of storage for an Integrated OpenShift Containerized Registry 
is very similar to configuration for regular GlusterFS persistent storage in 
that it can be either containerized or non-containerized. The following 
exceptions and additions apply:
```